webgui-new/package.json

```diff
@@ -13,7 +13,7 @@
     "@creditkarma/thrift-typescript": "^3.7.6",
     "@types/dagre": "^0.7.48",
     "@types/express": "^4.17.17",
-    "@types/react": "18.0.26",
+    "@types/react": "18.2.48",
```

Comment:
I don't know why this not given with a semantic version range in the first place. This should be updated later.